packit/api.py

```diff
@@ -625,9 +638,11 @@ def run_copr_build(
                     projectname=project,
                     chroots=chroots,
                     description=(
-                        "Continuous builds initiated by packit service.\n"
+                        description
+                        or "Continuous builds initiated by packit service.\n"
                         "For more info check out https://packit.dev/"
                     ),
+                    instructions=instructions,
```

Comment:
we should also have some default instructions, something like
```
You can check out the upstream project {upstream_project_url} to find out how to consume the builds.
```

Reply:
I just merged Jirka's PR for some other copr params, you'll need to rebase, sorry about that

I have an even better suggestion for the text:
```
You can check out the upstream project {upstream_project_url} to find out how to consume these builds. This copr project is created and handled by the packit project (https://packit.dev/).
```